mitodl/ocw-hugo-themes
# Use course title partial

#### What are the relevant tickets?
None

#### What's this PR do?
Fixes an issue with course titles I noticed. If you look at the page title for https://ocwnext.odl.mit.edu/courses/21h-244j-imperial-and-revolutionary-russia-culture-and-politics-1700-1917-fall-2019/ you see "MIT OpenCourseWare | Free Online Course Materials". This is incorrect, it should actually include the course title and possibly the section title or subsection titles. I wrote what you should see and where in the repro instructions for the PR that implemented it: https://github.com/mitodl/hugo-course-publisher/pull/327

I think there is a partial lookup issue but I'm not sure. It's unclear to me what happens if a page within `baseof/head.html` uses partial `title.html` and there are two `title.html` partials within our directory structure. The [hugo instructions](https://gohugo.io/templates/partials/#partial-template-lookup-order) seem vague on this. In any case the course partial seems to work for both cases so I just made it the only one.

#### How should this be manually tested?
Run `./build_scripts/build_all_courses.sh`. Verify that a course `index.html` file has a title within `<title>` tags starting with the course title, and verify that a section within that course has a `<title>` which starts with the section title.
